sql/moz-fx-data-shared-prod/telemetry_derived/crash_symbolicated_v1/query.sql

```diff
@@ -0,0 +1,100 @@
+WITH crash_frames AS (
+  SELECT
+    * REPLACE (FORMAT("0x%x", module_offset) AS module_offset)
+  FROM
+    `moz-fx-data-shared-prod`.telemetry_derived.crash_frames_v1
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+),
+crash_symbols AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod`.telemetry_derived.crash_symbols_v1
+  WHERE
+    submission_date = @submission_date
+  QUALIFY
+    -- symbols may be appended over multiple attempts, so only use the most recent symbols
+    0 = ROW_NUMBER() OVER (
```

Comment:
Can row number returned by row_number() be 0? According do this it is 1 based:

https://cloud.google.com/bigquery/docs/reference/standard-sql/numbering_functions#row_number

Reply:
```suggestion
    1 = ROW_NUMBER() OVER (
```